rs/tests/common.bzl

```diff
@@ -328,6 +328,8 @@ BOUNDARY_NODE_GUESTOS_RUNTIME_DEPS = [
     "//ic-os/boundary-guestos:scripts/build-bootstrap-config-image.sh",
 ]
 
+CLONER_CANISTER_RUNTIME_DEPS = ["//rs/tests:src/cloner_canister.wasm.gz"]
```

Comment:
Could `cloner_canister.wasm.gz` be moved under `rs/tests/networking`? We plan to remove the whole `/rs/tests/src` library in the context of [IDX-3144](https://dfinity.atlassian.net/browse/IDX-3144).